skipruntime-ts/adapters/postgres/src/index.ts

```diff
@@ -77,7 +78,9 @@ export class PostgresExternalService implements ExternalService {
    *
    * @param instance - Instance identifier of the external resource.
    * @param resource - Name of the PostgreSQL table to expose as a resource.
-   * @param params - Parameters of the external resource; **must** include a field `key` whose value is an object with a string field `col` identifying the table column that should be used as the key in the resulting collection, and a field `type` whose value is a Postgres text, integer, or serial type. (i.e. one of `TEXT`, `SERIAL`, `SERIAL2`, `SERIAL4`, `SERIAL8`, `BIGSERIAL`, `SMALLSERIAL`, `INTEGER`, `INT`, `INT2`, `INT4`, `INT8`, `BIGINT`, or `SMALLINT`)
+   * @param params - Parameters of the external resource; **must** include a field `key` describing the Postgres column that should be used as the key in the resulting collection
```

Comment:
Isn't everything following the `;` now redundant with the `params.key` entry below?

Reply:
Yep -- I have a diff to follow with more docusaurus changes, too.